doc/rbd/rbd-persistent-write-back-cache.rst

```diff
@@ -48,7 +48,11 @@ need to be enabled.::
         rbd plugins = pwl_cache
 
 Value of {cache-mode} can be ``rwl`` or ``ssd``. By default it is
-``disabled``
+``disabled``.
+
+The ``pwl_cache`` plugin depends on libpmem. This is only available on
+the x86_64 architecture on RHEL 8 and x86_64 and ppc64le on Fedora. It is not
+available on s390x.
```

Comment:
Can we be more generic here? I.e. you are mentioning RHEL and Fedore, but what is about other distros like Ubuntu or OpenSUSE? Is it necessary to mention distros at all? Could we just write that "this is not available on all architectures". It would give enough hint for the user and not become outdated in the future.

Reply:
@ktdreyer You seemed to agree with Mykola, want to make the change?